.github/workflows/esti.yaml

```diff
@@ -27,9 +27,21 @@ jobs:
         run: |
           echo "Enable next jobs based on secrets existence: ${{ env.ENABLE_NEXT_JOBS != '' }}"
           echo "secretsavailable=${{ env.ENABLE_NEXT_JOBS != '' }}" >> $GITHUB_OUTPUT
+  
+  extract-version:
```

Comment:
How do we extract version before we checkout the code?

Reply:
Magic 🎩🪄✨😊  
Fixed.